modules/mirror-registry-ssl-cert-replace.adoc

```diff
@@ -0,0 +1,58 @@
+// module included in the following assembly:
+//
+// * installing-mirroring-creating-registry.adoc
+
+:_content-type: PROCEDURE
+[id="mirror-registry-ssl-cert-replace_{context}"]
+= Replacing mirror registry for Red Hat OpenShift SSL/TLS certificates 
+
+In some cases, you might want to update their SSL/TLS certificates for the the _mirror registry for Red Hat OpenShift_. This is useful in the following scenarios:
+
+* If you are replacing the current _mirror registry for Red Hat OpenShift_ certificate.
+* If you are using the same certificate as the previous _mirror registry for Red Hat OpenShift_ installation. 
+* If you are periodically updating the _mirror registry for Red Hat OpenShift_ certificate. 
+
+Use the following procedure to replace _mirror registry for Red Hat OpenShift_ SSL/TLS certificates.
+
+.Procedure
+
+. Enter the following command to install the _mirror registry for Red Hat OpenShift_:
++
+[source,terminal]
+----
+./mirror-registry install \
+--quayHostname <host_example_com> \
+--quayRoot <example_directory_name>
+----
++
+This installs the _mirror registry for Red Hat OpenShift_ to the `$HOME/quay-install` directory. 
+
+. Prepare a new Certificate Authority (CA) bundle and generate new `ssl.key` and `ssl.crt` key files. For more information, see link:https://access.redhat.com/documentation/en-us/red_hat_quay/3/html-single/deploy_red_hat_quay_for_proof-of-concept_non-production_purposes/index#introduction-using-ssl[Using SSL/TLS].
+
+. Optional: Assign `/$HOME/quay-install` an environment variable, for example, `QUAY`, by entering the following command:
++
+[source,terminal]
+----
+$ export QUAY=/$HOME/quay-install
+----
+
+. Copy the new `ssl.crt` file to the `/$HOME/quay-install` directory by entering the following command:
++
+[source,terminal]
+----
+$ cp ~/ssl.crt $QUAY/quay-config
+----
+
+. Copy the new `ssl.key` file to the `/$HOME/quay-install` directory by entering the following command:
++
+[source,terminal]
+----
+$ cp ~/ssl.key $QUAY/quay-config
+----
+
+. Restart the `QUAY` application pod by entering the following command:
++
+[source,terminal]
+----
+$ systemctl restart quay-app 
+----
```

Comment:
The steps of certificate update work for me.

But I think there is one thing  should be changed.
  
![Screenshot from 2023-10-07 15-16-19](https://github.com/openshift/openshift-docs/assets/58208478/d7ca81c4-bf4a-4c11-bd34-1cc860010513)

For every command, there is a copy button in the end.  When you click this button,  the whole string in the code block will be copied. That means the `$` will be copied too.  I have to remove `$` when I paste the copied string in my terminal. 
```
[whu@whu]$ $ systemctl restart quay-app
bash: $: command not found...
```
If we much offer this copy button,  we should remove all `$`.

Reply:
Oh, I found this copy button is not offered by our code. Maybe customer know what is the correct operation. 

Anyway, the content looks good to me.